scraper/tests/smoke/test_smoke.py

```
@@ -77,6 +76,11 @@ def test_providers(self):
                     res = func()
                     assert isinstance(res, list)
 
+        # Google はライブ抽出を無効化しており get_page_text を呼ばない（フォールバック固定）
```

Ruff (0.15.21)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)